app/gather/assets/apps/utils/paths.jsx

```diff
@@ -125,6 +125,13 @@ export const getMasksAPIPath = ({ id, ...params }) => {
   return buildAPIPath(GATHER_APP, 'masks', id, params)
 }
 
+/**
+ * Returns the API url to configure defined consumers
+ */
+export const getConsumerConfigAPIPath = (id) => {
+  return buildAPIPath(GATHER_APP, 'surveys', id, { format: '', action: 'consumers-config' })
```

Comment:
you don't need to indicate the format :wink: 
```suggestion
  return buildAPIPath(GATHER_APP, 'surveys', id, { action: 'consumers-config' })
```